includes/discovery/sensors/states/linux.inc.php

```diff
@@ -31,7 +31,7 @@
         }
         $value = snmp_get($device, $oid.$codec, '-Oqv');
 
-        if (!empty($value)) {
+        if (stripos($value, 'abled') !== false) {
```

Comment:
I can also modify the ```stipos()``` if statement to ```if ($value = 'enabled' || $value = 'disabled')``` if the goal was to have some tighter checking on the line.